src/helpers/App.php

```diff
@@ -675,52 +686,45 @@ public static function defaultLogTargets(): array
         // If the dispatcher is configured with flushInterval => 1, it could cause a PHP error if any log
         // targets haven’t been instantiated yet.
 
-        $targets = [];
-
         $isConsoleRequest = Craft::$app->getRequest()->getIsConsoleRequest();
 
-        if ($isConsoleRequest || Craft::$app->getUser()->enableSession) {
-            $generalConfig = Craft::$app->getConfig()->getGeneral();
-
-            $fileTargetConfig = [
-                'class' => FileTarget::class,
-                'fileMode' => $generalConfig->defaultFileMode,
-                'dirMode' => $generalConfig->defaultDirMode,
-                'includeUserIp' => $generalConfig->storeUserIps,
-                'except' => [
-                    PhpMessageSource::class . ':*',
-                ],
-            ];
-
-            if ($isConsoleRequest) {
-                $fileTargetConfig['logFile'] = '@storage/logs/console.log';
-            } else {
-                $fileTargetConfig['logFile'] = '@storage/logs/web.log';
-            }
+        // Only log console requests and web requests that aren't getAuthTimeout requests
+        if (!$isConsoleRequest && !Craft::$app->getUser()->enableSession) {
+            return [];
+        }
```

Comment:
I'm not 💯 what the purpose of this check is, but keeping it in place…I dug up the comment from https://github.com/craftcms/cms/commit/5a3f2781190ae0122dc9f46550161c3480a5c260#diff-8ddcc488c3dc025ab9e7d97a17dedba912ac12119f5f5f7a010e07b4bf981688L559